### PR TITLE
Avoid division by zero when computing slice spacing when there is only one slice

### PR DESCRIPTION
**Pull request by nlessmann**
The DICOM image builder iterates over all slices and computes the difference in slice origin between consecutive slices, and finally averages those to determine the slice spacing. This leads to a division by zero when there is only one slice, which does not cause any problems but results in a RuntimeWarning. This PR adds a check to avoid this warning and a test for reading a single slice.

### CODE REVIEW

File: panimg/image_builders/dicom.py
Comment (MikeOverkamp-diag) [inline panimg/image_builders/dicom.py:221]: Does this not always result in a tuple of length 1, since both `origin_diff` and `n_diffs` are ints now?

Comment (MikeOverkamp-diag) [reply panimg/image_builders/dicom.py:221]: Which probably also means we need more test cases to cover this, as tests are now passing.

Comment (MikeOverkamp-diag) [reply panimg/image_builders/dicom.py:221]: Ah that makes sense, thank you for clarifying.

**Review by MikeOverkamp-diag — APPROVED** [review]

**Comment by MikeOverkamp-diag** [comment]
@jmsmkn I appear to not have write access to this repo, so I cannot merge this PR.

**Comment by jmsmkn** [comment]
Sorry! Try now.

**Pull request merged**
